packages/platform-android/native_modules.gradle

```diff
@@ -237,12 +236,19 @@ class ReactNativeModules {
   }
 }
 
+
+/*
+ * Sometimes Gradle can be called outside of JavaScript hierarchy. Detect the directory
+ * where build files of an active project are located.
+ */
+def projectRoot = rootProject.projectDir
```

Comment:
For context: `rootProject` is what you give a name in `settings.gradle` - it's the umbrella in case you have different smaller sub-projects

https://github.com/facebook/react-native/blob/master/template/android/settings.gradle#L1

For `autolinking` and CLI, it doesn't matter - we always want the main project. But you will notice that `applyNativeModulesBuildGradle` expects `project` argument to handle such case.